ch2driver/pytpcc/tpcc.py

```diff
@@ -362,13 +365,19 @@ def executorFunc(clientId, TAFlag, driverClass, schema, preparedTransactionQueri
     schema = constants.CH2_DRIVER_SCHEMA["CH2"]
     analyticalQueries = constants.CH2_DRIVER_ANALYTICAL_QUERIES["HAND_OPTIMIZED_QUERIES"]
     load_mode = constants.CH2_DRIVER_LOAD_MODE["NOT_SET"]
+    load_format = constants.CH2_DRIVER_LOAD_FORMAT["JSON"]
     bulkload_batch_size = constants.CH2_DRIVER_BULKLOAD_BATCH_SIZE
     kv_timeout = constants.CH2_DRIVER_KV_TIMEOUT
 
     if args['ch2p']:
         schema = constants.CH2_DRIVER_SCHEMA["CH2P"]
     elif args['ch2pp']:
         schema = constants.CH2_DRIVER_SCHEMA["CH2PP"]
+    elif args['ch2ppf']:
```

Comment:
Is CH2PF supposed to be supported for CSV loading too?

Reply:
We will not have a CH2PF variant.